mozci/console/commands/push.py

```diff
@@ -87,6 +87,7 @@ class ClassifyCommand(Command):
         {--high-confidence=0.9 : High confidence threshold used to classify the regressions.}
         {--output= : Path towards a directory to save a JSON file containing classification and regressions details in.}
         {--show-intermittents : If set, print tasks that should be marked as intermittent.}
+        {--retrigger : If set, retrigger UNKNOWN regressions}
```

Comment:
Marco and I had chatted about whether or not this would be better as a cli flag, or as a brand new command. I think I'd mildly prefer a new command, but if Marco / Bastien are ok with the flag here, then that's fine with me too.

If we do go the new command route, I'd call it something like `retrigger-unclassified`, and it would basically just do the exact same thing as this current patch does, except it wouldn't dump the classification results to stdout.

Reply:
Oh, this sounds great @ahal. Do we do it now, or merge and refactor the code later, or wait for Marco/Bastien's review?

Reply:
I don't have a preference between the two options (flag vs new command), either wfm.

Reply:
@chichi012 is this conversation ready to resolve?